.ci.yaml

```diff
@@ -1383,6 +1383,7 @@ targets:
 
   - name: mac_ios_native_ui_tests_ios32
     builder: Mac_ios native_ui_tests_ios32
+    bringup: true
```

Comment:
What does bringup mean? Does it mean something different from, or in addition to, flaky?

Reply:
It means flaky: https://github.com/flutter/flutter/wiki/CI-Config-Migration-Guide

Reply:
See go/flutter-build-test-infra for the full discussion, but we migrated to `bringup` as we don't want to encourage having flaky tests

Reply:
Internal link to the comment: https://docs.google.com/document/d/1xXTh1B0xXsxXKXdVGY4MqajEym11j01Dj9RxZlK333o/edit?resourcekey=0-lSU3Yr5lwwuDn8dw2fWzeA&disco=AAAAHzn0jew

> The only "real" use case for this is whether or not this is a new enough test that we don't want it to red the build if it goes wrong, to make debugging/fixing it easier initially.
An established builder should ordinarily not get marked flaky.

We discussed this in the Flutter design review, but I did not capture the notes